### ISSUE TITLE
Re-design home page

### ISSUE THREAD

**Issue by ststimac** [sonarwhal/sonarwhal.com#327]
The original home page design no longer suits our needs and has drastically changed content-wise since the original mock up.

Below are 3 different options I landed on. @molant @alrra @qzhou1607 @sarvaje any feedback or preferences are welcome.

![home_121317-04](https://user-images.githubusercontent.com/18073131/33968642-e6250454-e01d-11e7-984b-f5648955cbb9.png)

![home_121317-05](https://user-images.githubusercontent.com/18073131/33968644-e638c0d4-e01d-11e7-8f5f-b1fd0d0344a3.png)

![home_121317-06](https://user-images.githubusercontent.com/18073131/33968645-e64dc420-e01d-11e7-9c3c-7fed921f2655.png)

**Comment by qzhou1607** [comment]
@ststimac The scan input boxes with icon seem more eye-catching, definitely my favorites. And I really like the change of `search documentation` instead of `search...`. Occasionally I see people dropping urls in the documentation search bar, I kept wanting to open an issue about it but kept forgetting. Glad it's addressed now 😄

**Comment by sarvaje** [comment]
I think I would like a mix of all of them.
I like the header in 2 (Including the narwhal), but with the `search documentation` in 3 and the scan and install section in 1.
I also like the narwhal in the footer, but if it is already in the header, then it is too much hehehehe.

Also, I don't know if it is intentional or not, but `changelog` is missing in 2 and 3, and I think users should have access to it.

**Comment by ststimac** [comment]
@sarvaje re: changelog, yes sorry, that's from an old header, I forgot to delete it, it's still in the menu under About.

**Comment by molant** [comment]
Some thoughts about the content for the homepage:

* Now that we have the online version maybe the CLI instructions don't need to take that much space
* We need to better promote what it does, maybe bumping up the 4 concepts?
* Maybe we can change the instructions to the shortest possible command: `npx sonarwhal https://example.com`, and say something like "Try it on your machine".
* I think we should change the current slogan we have "build better websites - lint the web forward") to something a bit more catchy. We can use some placeholder for now while we come out with something better

Re the designs:

* Option 3 is my favorite
* The white input box pops up more, I'd use that one
* The "Getting started" button in 3 👌

What about something like the following (structure wise, I just copy pasted parts using my incredible paint skills):

![image](https://user-images.githubusercontent.com/606594/34008048-ea1c7402-e0b8-11e7-8823-1a2ad9118c0a.png)
 
Basically:

1. 1 phrase sonarwhal description
1. Try the online version (and maybe link to local install?)
1. Why you should use it
1. Local install, this could be also the `npx` command or we can even remove this whole section if we combine it with the online version somehow or think it doesn't have sense any more now that the getting started guide has been rewritten
1. Links to documentation

Let me know if you want to have a meeting and discuss it more!

**Comment by ststimac** [comment]
Updated home page design: yay or nay? I'd like to start building this out Monday.

**Comment by ststimac** [comment]
Oh I suppose I should share the mockup :P 

![home_121517](https://user-images.githubusercontent.com/18073131/34063237-a8701412-e1a5-11e7-8f1f-7c9e535d6b1b.png)

**Comment by molant** [comment]
> Oh I suppose I should share the mockup :P

That's probably a good idea 🙄

I like it. What about a version with the icebergs in the bottom like option 3?

**Comment by poshaughnessy** [comment]
Hope you don't mind me popping up and asking, but: what's the thought behind removing the `npm install ...` lines from the homepage?

**Comment by ststimac** [comment]
@molant I want to keep Nellie at the top so you're not just hit with all copy. 

@poshaughnessy I may have misinterpreted @molant's feedback but the focus on the homepage is to get people to use the online scanner but direct them to the CLI documentation if that's the route they'd like to take. We can add the `npx sonarwhal` line back in though.

**Comment by ststimac** [comment]
I guess we're putting the homepage design on hold until folks agree on the design.

**Comment by molant** [comment]
Yes, with some people OOF I think it's best to wait until new years if we don't have a full agreement.

Thanks!

**Comment by ststimac** [comment]
Notes from a quick sync on the homepage messaging:

- Explain what sonarwhal is & place it so that a user doesn't have to scroll to reach that information 
- Online scanner will be focus but we'll still have CLI on homepage 
- Keep the 4 columns about why you should use sonarwhal but cut down copy length
- Add a gif to accompany CLI info
- More space/separation between sections
- Better integrate documentation links/calls to action with copy

**Comment by ststimac** [comment]
Here's a new mockup of the home page. 

![home_011718_v1](https://user-images.githubusercontent.com/18073131/35125995-89c034ae-fc60-11e7-8dbb-eac3bf476362.png)

**Comment by ststimac** [comment]
Also, not all the copy is updated/complete yet. Placeholders in spots right now.

**Comment by molant** [comment]
I would swap the `cli` and `why use sonarwhal` sections.

I like the red button we had in the previous mockups. Any reason why we can't have it back?

@sonarwhal/contributors thoughts?

I'll think about the copy for some of the sections.

**Comment by ststimac** [comment]
The red button was really contrasting with the brighter blue. I can find a red/brighter color that works though.

**Comment by alrra** [comment]
> I would swap the cli and why use sonarwhal sections.

:+1:

---

Also, I think the content of the sections should be more aligned one with each other:

![](https://user-images.githubusercontent.com/1223565/35126715-8d4f043a-fc63-11e7-861e-243057053d6e.png)

**Comment by ststimac** [comment]
@alrra I think most of that will line up, this was a fairly quick mockup so it's not 100% polished.

**Comment by alrra** [comment]
>  I think most of that will line up,

Ok. 👍

**Comment by poshaughnessy** [comment]
Just to say that @diekus and I like the new design too!

- Lighter backgrounds :+1:
- Call to action with URL bar closer to the top of the page :+1:
- Keeps the developer instructions :+1:
- Cute Nellie! :+1:

**Comment by molant** [comment]
Here goes some options for the copy for your feedback. Please let me know what you think and if you have other ideas please share them!

# Scan your website for errors with sonarwhal

**Comments**:
* Make the `Run scan` button more obvious. I like the contrast from the previous mock ups with the red

**Title:**

*Scan your website for errors with sonarwhal (current)
* See what you can do to make your site even better
* Scan your site and learn how to make it better
* Use sonarwhal on your site to have the best version

**Description:**

* sonarwhal is a linting tool that will help you with your site's accessibility, speed, security, and more by checking your code for best practices and common errors.

# Install the sonarwhal CLI on your machine

**Title:**

* Install the sonarwhal CLI on your machine (current)
* Use sonarwhal in your machine
* Install sonarwhal locally
* Unlock all sonarwhal features from the CLI/your machine
* Install sonarwhal in your machine
* Run sonarwhal from your CLI

**Description:**

* If you want to disable a certain rule, further configure another one, ignore domains, or more, run sonarwhal from your CLI.
* to unlock the full potential of sonarwhal, run it from the CLI
 
# Why use sonarwhal?

**Comments**:

* The titles of this entries are all upper case, in the next section they aren't. We should be consistent with the font and casing
* I think we should explain in this section what is sonarwhal. We mention it at the very beginning but maybe we should repeat a similar message + what we have.
* Will think about the 3-4 things we want to highlight and update once I have something. **Ideas are more than welcome**

**Title:**

* Why use sonarwhal? (current)
* What is sonarwhal?

# Links to docs

**Comments:**

* I think I prefer to have `guide` all lower cased. In the current live version we aren't consistent so let's fixed that.
* I think it should be `Rules` and not `Changelog`
* Need to think about the copy of each bullet

**Comment by ststimac** [comment]
> I think it should be Rules and not Changelog

I was thinking of changing that to Changelog because we'll have links to the rules in an above section.

**Comment by molant** [comment]
The section above only talks about creating custom rules, not the current list of supported rules.
It could also change depending on the new sections we put in there that might be an option.

**Comment by molant** [comment]
@sonarwhal/core, @ststimac, @poshaughnessy, @qzhou1607 what do you think of the copy proposed so far in https://github.com/sonarwhal/sonarwhal.com/issues/327#issuecomment-359528934 ?

**Comment by ststimac** [comment]
@molant I had a change or two but haven't gone through it all. I was mostly just adjusting some wording. Will look through everything tomorrow.

**Comment by molant** [comment]
OK, just saying because I proposed a few things for titles, etc. and I wanted to have others people opinion.

**Comment by ststimac** [comment]
### Current Title: Scan your website for errors with sonarwhal (current)
Proposed: Use sonarwhal to improve your website

Description: No change 

### Current Title: Install the sonarwhal CLI on your machine
Proposed: Run sonarwhal from your CLI

Description (Proposed): If you want to disable specific rules, further configure others, ignore domains, or more, run sonarwhal from your CLI

### Why Use Sonarwhal section

I think the title of this section will depend on how the messages change. Right now the subheadings all sort of answer "why use sonarwhal" but if the messages answer "what is sonarwhal" then we should update it.

**Comment by ststimac** [comment]
I also just realized that if we change the messaging for "Why use sonarwhal", it will probably affect the illustration I'm about to do for the "fully customizable" column. 

@molant @alrra thoughts?

**Comment by molant** [comment]
Fully customizable or something similar to that will still be there so I would worry too much. I might actually add more bullets so that section's design might have to change a bit.

**Comment by molant** [comment]
@kborchers we are redesigning the homepage. Do you want us to add something about the JS Foundation in the footer?  I remember talking about this a while ago but I can't remember the output.

**Comment by kborchers** [comment]
@molant We actually have a couple of things we ask projects to add to their sites before they can graduate from mentorship so adding them now would get that out of the way. You can see an example of both items on appium.io where there is a JSF project logo header and the copyright and trademark statement in the footer.

**Comment by alrra** [comment]
From a discussion with @ststimac and @molant: we should also move and make the GitHub link to this repository more evident on the main page.

**Comment by ststimac** [comment]
@alrra @molant would something like the following work for the social icons? (I don't have the medium icon handy but it would be up there too, also this is not to scale, just a quick mockup):

<img width="935" alt="screen shot 2018-01-30 at 4 40 41 pm" src="https://user-images.githubusercontent.com/18073131/35599208-80a8a8a8-05dc-11e8-958c-9051094e2445.png">

In the main mockup, I also added an additional link to the github under the "community driven" column.

**Comment by molant** [comment]
I'm not sure about the double header.
Maybe remove the Twitter logo and put GitHub/Source code in the top menu?

**Comment by kborchers** [comment]
@molant @ststimac IMO the double header works and we could create a light version of the logo for the JSF header I mentioned earlier to use that could fit on the left side of that header if that helps balance it

**Comment by ststimac** [comment]
@kborchers I'm open to trying that. Do you have a light version handy?

**Comment by kborchers** [comment]
@ststimac https://js.foundation/wp-content/uploads/sites/33/2018/01/JS-Foundation-Project-Tagline-white.png

**Comment by ststimac** [comment]
Quick screenshot:

![screen shot 2018-01-31 at 11 33 32 am](https://user-images.githubusercontent.com/18073131/35643266-9e8808b6-067a-11e8-8648-f11f88380ac2.png)
 
I think the addition of the JSF logo definitely helps to balance it out.

**Comment by alrra** [comment]
> https://js.foundation/wp-content/uploads/sites/33/2018/01/JS-Foundation-Project-Tagline-white.png

@kborchers Do you have a SVG version of that? Thanks!

**Comment by kborchers** [comment]
@alrra https://js.foundation/wp-content/uploads/sites/33/2018/02/JS_Foundation_Project_Tagline_White.svg

**Comment by ststimac** [comment]
@molant @alrra for the CLI gif, what should that show? All the way through a scan?

**Comment by molant** [comment]
Or maybe just the installation and creation of the config file.
We are changing a few parts of the process [this PR](https://github.com/sonarwhal/sonarwhal/pull/844), that why we haven't done one yet.

If you have any ideas of place holders for now go for it.

**Comment by molant** [comment]
Also it should be a video with no sound and not autoplay to make it accessible.
